lib/rownd_cubit.dart

```diff
@@ -61,11 +61,7 @@ class RowndCubit extends Cubit<AuthState> {
       return;
     }
 
-    user = {
-      "firstName": userData['first_name'],
-      "lastName": userData['last_name'],
-      "email": userData['email'],
-    };
+    user = userData;
```

Comment:
I think as an API consumer, I would expect this function to also return the user data.

Reply:
It's a private function so they won't have access to use the function. They just have access to `user` property.